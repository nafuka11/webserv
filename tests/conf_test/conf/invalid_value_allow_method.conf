server {
    allow_method HEAD;
}