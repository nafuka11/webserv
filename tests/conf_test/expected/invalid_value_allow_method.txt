webserv: invalid value in "allow_method" directive in ./conf/invalid_value_allow_method.conf:2
